### DISCUSSION TITLE
Can I apply a modifier to a list of terms?

### DISCUSSION THREAD

**Discussion by ebeauchamps** [statamic/cms#9572] [Q&A] [answered]
This question is another take at a discussion started on Discord. But I don't feel it's a cross-post...I need more infos.

I have a "tags" taxonomy linked to an "articles" collection
Each article can have multiple terms (ie Paris, Nantes, Rennes)
*Inside* a collections:articles tag pair, I want each article to display terms that are associated with it.

so I have a markup as <p>{{ tags }} {{ title }} {{ /tags }}</p> but I cannot get the 3 terms to display nicelly in the same paragraph

I would like to use {{ title | sentence_list }} or {{ title | ampersand_list }} but to no avail.

Is this a feature request?

PS: I also tried this: <p>{{ tags }} {{ terms }} {{ title }} {{ /terms }} {{ /tags }}</p>

**Comment by duncanmcclean** [comment, marked as answer]
Would something like this work?

```antlers
{{ categories | pluck('title') | sentence_list }}
```

**Reply by ebeauchamps** [reply]
oh. Very nicely indeed. Thanks sir, better markup than mine.
